clients/cli/src/clap_app.rs

```diff
@@ -2688,4 +2700,56 @@ pub fn app<'a>(
                 .arg(multisig_signer_arg())
                 .nonce_args(true)
         )
+        .subcommand(
+            SubCommand::with_name(CommandName::Pause.into())
+                .about("Pause mint, burn, and transfer")
+                .arg(
+                    Arg::with_name("token")
+                        .validator(|s| is_valid_pubkey(s))
+                        .value_name("TOKEN_MINT_ADDRESS")
+                        .takes_value(true)
+                        .index(1)
+                        .required(true)
+                        .help("The pausable token address"),
+                )
+                .arg(
+                    Arg::with_name("pause_authority")
+                    .long("pause-authority")
```

Comment:
nit: do you want to alias this to `owner` like with freeze / thaw?
```suggestion
                    Arg::with_name("pause_authority")
                    .long("pause-authority")
                    .alias("owner")
```

Reply:
Yep, let's do that for consistency.